clang/include/clang/Driver/Options.td

```diff
@@ -2489,7 +2489,7 @@ def fsycl_instrument_device_code : Flag<["-"], "fsycl-instrument-device-code">,
   HelpText<"Add ITT instrumentation intrinsics calls">,
   MarshallingInfoFlag<CodeGenOpts<"SPIRITTAnnotations">>;
 defm sycl_id_queries_fit_in_int: BoolFOption<"sycl-id-queries-fit-in-int",
-  LangOpts<"SYCLValueFitInMaxInt">, DefaultFalse,
+  LangOpts<"SYCLValueFitInMaxInt">, DefaultTrue,
```

Comment:
I'm not up on the options rewrite, is there a '-fno-...' flag?

Reply:
yes, there is fno flag: -fno-sycl-id-queries-fit-in-int

Reply:
BoolFOption covers both variants (`-fsycl-id-queries-fit-in-int` and `-fno-sycl-id-queries-fit-in-it`)